### PR TITLE
ToCode Solution: תרגול מודולים

### PR DESCRIPTION
**Pull request by EyalAbel**

### CODE REVIEW

File: 06_modules_lab/01.py [hunk 1]
```diff
@@ -1,4 +1,26 @@
...
+
+
+def Usage():
+    print ('Invalud arguments.\nUsage: 01.py N')
+########################################################################################################################
+#
+#
+def main():
+    if len(sys.argv) != 2:
+        Usage()
```
Comment (ynonp) [inline 06_modules_lab/01.py:15]: Prefer lowercase for function names (such as `len`, `sys.argv`, `sys.exit`)

File: 06_modules_lab/03_bonus.py [hunk 2]
```diff
@@ -1,13 +1,40 @@
...
+#
+def main():
+    path  = "."
+    parser = argparse.ArgumentParser(description='ArgumentParser use exercise')
+    parser.add_argument("-path", help="path for the working folder")
+    args = parser.parse_args()
+    if args.path:
+        path = args.path
+        if not os.path.isdir(path):
+            print "Error: path must be a valid (existing) directory"
```
Comment (ynonp) [inline 06_modules_lab/03_bonus.py:32]: Probably better to sys.exit() at this point

File: 06_modules_lab/03_bonus.py [hunk 3]
```diff
@@ -1,13 +1,40 @@
...
+    parser = argparse.ArgumentParser(description='ArgumentParser use exercise')
+    parser.add_argument("-path", help="path for the working folder")
+    args = parser.parse_args()
+    if args.path:
+        path = args.path
+        if not os.path.isdir(path):
+            print "Error: path must be a valid (existing) directory" 
+    size = 1024 * 1024 # 1 Megabyte
+    print_file_by_size(size,path)
+    sys.exit()
```
Comment (ynonp) [inline 06_modules_lab/03_bonus.py:35]: Why do you need to sys.exit here?

File: 06_modules_lab/03_bonus.py [hunk 4]
```diff
@@ -1,13 +1,40 @@
...
+def main():
+    path  = "."
+    parser = argparse.ArgumentParser(description='ArgumentParser use exercise')
+    parser.add_argument("-path", help="path for the working folder")
+    args = parser.parse_args()
+    if args.path:
+        path = args.path
+        if not os.path.isdir(path):
+            print "Error: path must be a valid (existing) directory" 
+    size = 1024 * 1024 # 1 Megabyte
```
Comment (ynonp) [inline 06_modules_lab/03_bonus.py:33]: `size` should also be provided via args

File: 06_modules_lab/03_bonus.py [hunk 5]
```diff
@@ -1,13 +1,40 @@
...
-
+#!/usr/bin/env python
+
+__author__ = 'Eabel'
+
+import sys
+import os
+import argparse
+
+def Usage():
```
Comment (ynonp) [inline 06_modules_lab/03_bonus.py:9]: `argparse` already prints a nice help + usage message when script is used incorrectly

Comment (EyalAbel) [inline 06_modules_lab/03_bonus.py:9]: I didn't use this function (kept it from previous exercise).

File: 06_modules_lab/03_bonus.py (hunk 3 above)
Comment (EyalAbel) [inline 06_modules_lab/03_bonus.py:35]: Do real need, just to be explicit that this is the exit point.

**Pull request closed without merging**